pkgs/build-support/fetchurl/mirrors.nix

```
@@ -73,7 +73,9 @@ rec {
   # Mirrors of ftp://ftp.kde.org/pub/kde/.
   kde = [
     "http://download.kde.org/download.php?url="
```

Could we also do
```suggestion
    "https://download.kde.org/download.php?url="
```

Note that it's a redirect anyway:
```shell
> http -h http://download.kde.org/download.php
HTTP/1.1 301 Moved Permanently
[...]
Location: https://download.kde.org/download.php
```